docs/Introduction/getting-started/deploy-your-first-contract.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: nodes.liquid
+section: gettingStarted
+date: Last Modified
+title: "Deploy Your First Smart Contract"
```

Comment:
We use deploy multiple ways in our docs. We "deploy to remix" as well in earlier steps. Should we rename all of our remix buttons to something else?

Reply:
Standardized the buttons to say "Open in Remix" except in cases where custom button action text is used.